docs/ios/data-cloud/introduction-to-icloud.md

```diff
@@ -412,8 +412,8 @@ void LoadDocument (NSMetadataQuery metadataQuery)
 
 ### Displaying iCloud Documents
 
-Displaying a UIDocument shouldn't be any different to any other model class
-- properties are displayed in UI controls, possibly edited by the user and
```

Comment:
I think this was just because of text wrapping rather than an intentional list item